### PR TITLE
Show attachments on employee application view

### PR DESCRIPTION
**Pull request by reynders**
#### Checklist for pull request creator

- [ ] The code is consistent with the existing code base
- [ ] Tests have been written for the change (e2e, integration tests)
- [ ] The change has been tested locally
- [ ] The change conforms to the UX specifications
- [ ] The code is self-documenting or has been documented sufficiently, e.g., in the README
- [ ] The branch has been rebased against master before the PR was created

#### Checklist for pull request reviewer (copy to review text box)

```
- [ ] All changes in all changed files have been reviewed
- [ ] The code is consistent with the existing code base
- [ ] Tests have been written for the change (e2e, integration tests)
- [ ] The change has been tested locally
- [ ] The change conforms to the UX specifications
- [ ] The code is self-documenting or has been documented sufficiently, e.g., in the README
- [ ] The PR branch has been rebased against master and force pushed if necessary before merging
```

### CODE REVIEW

**Review by hruoho — APPROVED** [review]
This was suggested: 
```
attachments = row.mapJsonColumn<Array<Attachment>>("attachments").toList()
```
but I can fix it in the follow up PR

File: frontend/packages/enduser-frontend/src/config.ts [hunk 1]
```diff
@@ -221,7 +221,7 @@ configs.prod = defaultsDeep(
       daycareApplication: true,
       preschoolApplication: true,
       selectApplicationType: true,
-      attachments: false
+      attachments: true
```
Comment (jaahon) [inline frontend/packages/enduser-frontend/src/config.ts:224]: NOOO

**Review by jaahon — CHANGES_REQUESTED** [review]

File: frontend/packages/enduser-frontend/src/config.ts [hunk 2]
```diff
@@ -179,7 +179,7 @@ configs._default = {
     daycareApplication: true,
     preschoolApplication: true,
     selectApplicationType: true,
-    attachments: true
+    attachments: false
```
Comment (jaahon) [inline frontend/packages/enduser-frontend/src/config.ts:182]: @hruoho this wasn't necessary

Comment (hruoho) [reply frontend/packages/enduser-frontend/src/config.ts:182]: i'm lost

Comment (jaahon) [reply frontend/packages/enduser-frontend/src/config.ts:182]: 😂

Comment (jaahon) [reply frontend/packages/enduser-frontend/src/config.ts:182]: You changed it so that attachments are hidden _by default_ instead of just in _prod_.

Comment (hruoho) [reply frontend/packages/enduser-frontend/src/config.ts]: I see :facepalm:

**Review by jaahon — APPROVED** [review]

**Pull request merged**